### PR TITLE
Add mentorship section to template + editor guide

### PR DESCRIPTION
**Pull request by noamross**
This adds a section to the request template specific to new reviewers who have requested mentorship, and initiates some guidance on providing that mentorship.  It is draft as there are two things to consider.

1) @yabellini has been collecting materials for peer-review mentorship.  Could you point us to them? It would be useful for editors to have them on hand for new reviewers, and link to them from the guide for editors.

2) Spanish translation: What is the preferred current workflow for adding Spanish to a PR? Should I do a DeepL translation manually to include in the `*.es` files?

### CODE REVIEW

**Comment by yabellini** [comment]
>     1. @yabellini has been collecting materials for peer-review mentorship.  Could you point us to them? It would be useful for editors to have them on hand for new reviewers, and link to them from the guide for editors.

@noamross  Do you mean the training material I made for the mentors training for the champions program?

**Comment by noamross** [comment]
@yabellini Possibly. @maurolepore told me about this.  Primarily I'm looking to have a collection of items such as high-quality reviews or descriptions of the process that editors can point new reviewers to. We have some of this and can create it, but if some is already aggregated for the Champions program we can make use of it.

**Comment by yabellini** [comment]
For the champion program, the material is in mentoring in general, not for peer review in particular. 
We will publish on the website everything soon, I'm working on it. 
The kind of material you mention would be useful for the champions program too.

File: softwarereview_editor.es.Rmd [hunk 1]
```diff
@@ -78,11 +78,13 @@ Tus herramientas para buscar personas para la revisión en *Airtable* son
 
 ![Captura de pantalla de la interfaz de filtros de Airtable con un filtro sobre conocimientos técnicos que tiene que incluir química y en áreas técnicas tienen que incluir integración continua (interfaz y etiquetas en inglés)](images/airtable.png)
 
-- crear una [visualización duplicada](https://support.airtable.com/hc/en-us/articles/202624989-Guide-to-views), dependiendo de tus conocimientos de Airtable.
+- crear una [visualización personalizada](https://support.airtable.com/hc/en-us/articles/202624989-Guide-to-views) dependiendo de tus habilidades en Airtable.
 
 Echa un vistazo a los campos `last_time_contacted` (último contacto) y `last_answer` (última respuesta) antes de ponerte en contacto con alguien.
 Si alguien se ha negado recientemente porque no tenía tiempo, podría ser mejor abstenerse, mientras que no hace falta esperar demasiado para volver a invitar a alguien que se negó debido a conflictos de interés.
 
+Además, comprueba si un revisor novel ha indicado que `require_mentorship`.  En caso afirmativo, utiliza la parte de tutoría de la plantilla de correo electrónico y prepárate para proporcionar orientación adicional.
```
Comment (yabellini) [inline softwarereview_editor.es.Rmd:86]:
```suggestion
Además, comprueba si una persona que es nueva revisando ha indicado que `require_mentorship`.  En caso afirmativo, utiliza la parte de tutoría de la plantilla de correo electrónico y prepárate para proporcionar orientación adicional.
```

File: templates/request.es.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ Nuestro objetivo es que las revisiones sean abiertas, no conflictivas y centrada
 ¡Sé amable!
 Consulta nuestra guía de revisión y el [código de conducta](https://ropensci.org/code-of-conduct/) para más información.
 
+[SI SE SOLICITÓ TUTORÍA: Has indicado en tu formulario que preferirías tutoría para tu primera revisión.  
+Eres libre de utilizarme como recurso durante este proceso, lo que incluye hacer preguntas por correo electrónico y Slack (recibirás una invitación al Slack de rOpenSci), y compartir borradores de revisiones para recibir devoluciones antes de publicarlas.
```
Comment (yabellini) [inline templates/request.es.md:17]:
```suggestion
Sientete libre de contactarme cuando lo necesites durante este proceso, lo que incluye hacer preguntas por correo electrónico y Slack (recibirás una invitación al Slack de rOpenSci), y compartir borradores de revisiones para recibir devoluciones antes de publicarlas. 
```

File: templates/request.es.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ Nuestro objetivo es que las revisiones sean abiertas, no conflictivas y centrada
 ¡Sé amable!
 Consulta nuestra guía de revisión y el [código de conducta](https://ropensci.org/code-of-conduct/) para más información.
 
+[SI SE SOLICITÓ TUTORÍA: Has indicado en tu formulario que preferirías tutoría para tu primera revisión.  
+Eres libre de utilizarme como recurso durante este proceso, lo que incluye hacer preguntas por correo electrónico y Slack (recibirás una invitación al Slack de rOpenSci), y compartir borradores de revisiones para recibir devoluciones antes de publicarlas. 
+También estaré encantada de hacer una breve llamada Zoom para explicar el proceso.Por favor, házmelo saber en tu respuesta si quieres programar una.]
```
Comment (yabellini) [inline templates/request.es.md:18]:
```suggestion
También me encantará hacer una breve videollamada para explicar el proceso. Por favor, házmelo saber en tu respuesta si quieres programar una.]
```

File: templates/request.es.md [hunk 4]
```diff
@@ -13,6 +13,10 @@ Nuestro objetivo es que las revisiones sean abiertas, no conflictivas y centrada
...
+
 ¿Puedes revisar? Si no puedes, siempre nos sirve si puedes sugerir personas alternativas. Si no tengo noticias tuyas en el plazo de una semana, supondré que no puedes hacer una revisión en este momento.
```
Comment (yabellini) [inline templates/request.es.md:20]:
```suggestion
¿Puedes revisar? Si no puedes, siempre nos sirve si puedes sugerir a otras personas. Si no tengo noticias tuyas en el plazo de una semana, supondré que no puedes hacer una revisión en este momento.
```

File: templates/request.es.md [hunk 5]
```diff
@@ -13,6 +13,10 @@ Nuestro objetivo es que las revisiones sean abiertas, no conflictivas y centrada
 ¡Sé amable!
 Consulta nuestra guía de revisión y el [código de conducta](https://ropensci.org/code-of-conduct/) para más información.
 
+[SI SE SOLICITÓ TUTORÍA: Has indicado en tu formulario que preferirías tutoría para tu primera revisión.
```
Comment (yabellini) [inline templates/request.es.md:16]:
```suggestion
[SI SE SOLICITÓ TUTORÍA: Has indicado en tu formulario que te gustaría tener tutoría para tu primera revisión.  
```

File: templates/request.md [hunk 6]
```diff
@@ -13,6 +13,11 @@ We aim for reviews to be open, non-adversarial, and focused on improving softwar
 Be respectful and kind! 
 See our reviewers guide and [code of conduct](https://ropensci.org/code-of-conduct/) for more.
 
+[IF MENTORSHIP REQUESTED: You indicated in your form that you'd prefer mentorship for your first review.  
+You are free to use me as a resource during this process, including asking questions by email and Slack (you will receive an invite to the rOpenSci Slack), and sharing draft reviews for feedback before posting. 
+I'm also happy to have a brief Zoom call to walk through the process.
```
Comment (yabellini) [inline templates/request.md:18]:
```suggestion
I'm also happy to have a brief videocall to walk through the process.  
```

**Review by yabellini — APPROVED** [review]
Looks very nice! Small changes on the Spanish one.  I removed Zoom and used videocall in case people like to use another software.

**Pull request merged**